ros2_batch_job/__main__.py

```diff
@@ -85,8 +84,9 @@ def get_args(sysargv=None):
         help='create an archive of the install space')
     parser.add_argument(
         '--repo-file-url',
-        default=DEFAULT_REPOS_URL,
-        help="url of the ros2.repos file to fetch and use for the basis of the batch job")
+        help="url of the ros2.repos file to fetch and use for the basis of the batch job",
+        required=True)
```

Comment:
Correct. I [moved](e0d0edcc87898412d35acad806cfbe52cbe1296e) the `required` attribute before `help` since I find is easier to read when the "important" attributes come first.